src/main/resources/data/mappingProfiles/defaultMappingProfiles.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "25d81cbe-9686-11ea-bb37-0242ac130002",
+  "name": "Default instance mapping profile",
+  "description": "Default mapping profile for the inventory instance record",
+  "recordTypes": [
+    "INSTANCE"
+  ],
+  "outputFormat": "MARC",
+  "userInfo": {
+    "firstName": "DIKU",
+    "lastName": "ADMINISTRATOR",
+    "userName": "diku_admin"
+  },
+  "metadata": {
+    "createdDate": "2020-05-15T09:39:13.659+0000",
+    "createdByUserId": "08d3ea27-e4ff-4a26-83e1-d5f86f16495c",
```

Comment:
Let's not add the userId, it will not be available in all the environments

Reply:
@KVupp looks like there is a validation in RMB, and an empty string is not pass through the regex

Reply:
i will remove createdByUserId and updatedByUserId fields from metadata then